Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.py

```diff
@@ -211,41 +211,41 @@ def _get_writeback_data(self):
     def _call(self, url_suffix, **kwargs):
 
         json_data = {
-            'demisto_command': demisto.command(),
-            'demisto_args': demisto.args(),
+            "demisto_command": demisto.command(),
+            "demisto_args": demisto.args(),
         }
 
         request_kwargs = {
-            'method': 'post',
-            'url_suffix': url_suffix,
-            'json_data': json_data,
-            'timeout': 90,
-            'retries': 3,
-            'status_list_to_retry': STATUS_TO_RETRY,
+            "method": "post",
+            "url_suffix": url_suffix,
+            "json_data": json_data,
+            "timeout": 90,
+            "retries": 3,
+            "status_list_to_retry": STATUS_TO_RETRY,
         }
 
         request_kwargs.update(kwargs)
 
         # This need to be after 'request_kwargs.update(kwargs)'.
         calling_context = self._get_writeback_data()
         if calling_context:
-            request_kwargs['json_data']['callingContext'] = calling_context
+            request_kwargs["json_data"]["callingContext"] = calling_context
```

Comment:
Here and all places in the code with direct access [],
if the key does not exist, will be thrown an exception of KeyError, is that ok for you?

Reply:
I reviewed all the places where we access directly by key, and all actually safe from KeyError or exception handler is already in place. Will update 1 place where it could be better.